src/overview/container.js

```diff
@@ -109,6 +113,9 @@ class OverviewContainer extends Component {
                 {...this.props}
                 setInputRef={this.setInputRef}
                 onInputChange={this.handleInputChange}
+                onShowFilterChange={this.props.onShowFilterChange}
```

Comment:
These props should all be included in the `{...this.props}` restructuring. So not entirely necessary here.